src/eslint-config-adeira/runner/run.js

```diff
@@ -33,16 +33,28 @@ type Options = {|
 // TODO: we should eventually get rid of 'create-jest-runner' dependency and use our own
 //  implementation with this built-in mechanism. This way we don't have to iterate all the files
 //  and skipping them.
-const changedFiles = Git.getChangesToTest();
+const getChangedFiles = () => {
+  try {
+    // This fails if git is not initialised or if user has no remote origin
+    // Let's catch the error, and just lint all instead rather than crashing
+    return Git.getChangesToTest();
+  } catch {
+    return null;
```

Comment:
We could also return an empty array `[]` to simplify the conditions bellow. It's strange that `changedFiles` can be an array but sometimes also a `null`. I'd prefer to annotate this `getChangedFiles` function to always return `$ReadOnlyArray<string>`

Reply:
I thought about this, it would lead to `let runAll = extraOptions.runAll || changedFiles.length === 0;`

Meaning you would lint all if you didn't change any files